### DISCUSSION TITLE
Help with setup on my Local Pc

### DISCUSSION THREAD

**Discussion by asadshoyebkhan** [jmateoac/plasma#193] [General]
How to setup this project on my local Pc and stated working on issues

**Comment by jmateoac** [comment]
Hey @asadshoyebkhan! 
I'm more than happy to help you get set up.

Do you already have Android Studio? If not, you can download the Preview version at https://developer.android.com/studio/preview.
Either Beta or Canary will be OK (I'm using Canary myself)

Once you download it, you should be able to open the folder and build the app as usual. Let me know if you need any help. I haven't set up a new environment in a while, so there's a possibility that I missed a dependency or two.
